perun-core/src/test/java/cz/metacentrum/perun/core/entry/MembersManagerEntryIntegrationTest.java

```diff
@@ -88,6 +88,37 @@ public void setUp() throws Exception {
 	}
 
 	@Test
+	public void findSponsoredMembers() throws Exception {
```

Comment:
Test name should conform tested method name. In this case ``getSponsoredMembers`` (same for println below).